library/src/main/java/eu/inloop/viewmodel/ViewModelHelper.java

```diff
@@ -145,8 +145,8 @@ public void onDestroyView(@NonNull Fragment fragment) {
             return;
         }
         mViewModel.clearView();
-        if (fragment.getActivity() != null && fragment.getActivity().isFinishing()) {
-            removeViewModel(fragment.getActivity());
+        if (fragment.requireActivity().isFinishing()) {
```

Comment:
this is a change in behaviour,  we need to keep `fragment.getActivity() != null && fragment.getActivity().isFinishing()` to prevent crashes

Reply:
I thought that activity shouldn't be null and you just check if activity is finishing. I will rewrite it.